webpack/assets/javascripts/react_app/components/HostStatuses/HostStatusesSelectors.js

```diff
@@ -0,0 +1,74 @@
+import { selectAPIResponse } from '../../redux/API/APISelectors';
+import {
+  GLOBAL_STATUS_OK,
+  GLOBAL_STATUS_WARN,
+  GLOBAL_STATUS_ERROR,
+  HOST_STATUSES_KEY,
+} from './HostStatusesConstants';
+
+export const selectHostStatuses = state =>
+  selectAPIResponse(state, HOST_STATUSES_KEY)?.results || [];
+
+export const selectHostStatusesNames = state =>
+  selectHostStatuses(state).map(({ name }) => name);
+
+export const selectHostStatus = (state, statusName) =>
+  selectHostStatuses(state).find(({ name }) => name === statusName);
+
+export const selectHostStatusDetails = (state, statusName) =>
+  selectHostStatus(state, statusName)?.details || [];
+
+export const selectHostStatusDescription = (state, statusName) =>
+  selectHostStatus(state, statusName)?.description || '';
+
+/* eslint-disable camelcase */
+export const selectHostStatusOkTotalPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.ok_total_path;
+
+export const selectHostStatusOkOwnedPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.ok_owned_path;
+
+export const selectHostStatusWarnTotalPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.warn_total_path;
+
+export const selectHostStatusWarnOwnedPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.warn_owned_path;
+
+export const selectHostStatusErrorTotalPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.error_owned_path;
+
+export const selectHostStatusErrorOwnedPath = (state, statusName) =>
+  selectHostStatus(state, statusName)?.error_total_path;
```

Comment:
How about some aggregation?

```js
export const selectHostStatusTotalPath = () => ({
  okTotalPath: selectHostStatus(state, statusName)?.ok_total_path,
  warbTotalPath: selectHostStatus(state, statusName)?.warn_total_path,
  // ...etc
})

export const selectHostStatusOwnedPath = () => ({
  okOwnedlPath: selectHostStatus(state, statusName)?.ok_owned_path;,
  warnOwnedPath: sselectHostStatus(state, statusName)?.warn_owned_path;,
  // ...etc
})
```
This locally aggregate the data and saves some redundant imports later